core/src/utils/stores.ts

```diff
@@ -248,6 +264,12 @@ export const writablesForProps = <T extends object>(
 	return [stores, createPatch(stores)];
 };
 
+/**
+ * Using input stores, builds an object containing the stores as readable and a global state.
```

Comment:
Maybe `Using input stores, this function builds an object containing the stores as readable and a global state.`